mitodl/ol-infrastructure
# Pydantic v2 round2

# What are the relevant tickets?
N/A

# Description (What does it do?)
Upgrades Pydantic usage to be compatible with the version 2 release

# How can this be tested?
- Ensure that a Concourse pipeline renders properly, including all of the `step` elements by executing the appropriate Python module in `ol_concourse`
- Ensure that PyInfra builds are rendered properly, particularly configuration files
- Ensure that component resources in Pulumi build properly
